### ISSUE TITLE
`store-backup/index` looks odd

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#394]
derived from https://github.com/kazurayam/inspectus/issues/66

I could create the `store-backup/index.html` file, but it looks odd.

<img width="877" alt="スクリーンショット 2022-12-15 23 10 13" src="https://user-images.githubusercontent.com/4467417/207884079-3d81673c-eea5-4090-af6e-29bf5b773c0b.png">

**Comment by kazurayam** [comment]
In the https://github.com/kazurayam/materialstore/blob/0.14.1/src/main/java/com/kazurayam/materialstore/base/report/IndexCreator.java

```
    public Path create() throws MaterialstoreException, IOException {
        /* create a data-model */
        ...
        dataModel.put("title", "store/index.html");
        ReportFileList rfl = new ReportFileList(store);
        dataModel.put("model", rfl.toTemplateModel());
```

A static text "store/index.html" is given. This is poorman's band-aid.

**Comment by kazurayam** [comment]
fixed at v0.14.2

![Uploading スクリーンショット 2022-12-16 9.14.38.png…]()
